lib/decoders/transform.js

```diff
@@ -1,3 +1,5 @@
+
+import _ from 'underscore';
```

Comment:
Do you know how lodash differs from underscore?  I've heard people say lodash is underscore but better, essentially, and I think the Frontend might use lodash.